.travis.yml

```diff
@@ -2,7 +2,7 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
+php: [5.3, 5.4, 5.5, 5.6, 7.1, hhvm]
```

Comment:
Please return PHP 7.0 back and of course keep PHP 7.1.

---

I see no reasons for dropping PHP 7.0 support.